### PR TITLE
Fix C++ for parasymbolic so that it compiles

### PR DESCRIPTION
**Pull request by benglewis**

### CODE REVIEW

File: src/corona_hakab_model/parasymbolic_matrix/parasymbolic.cpp [hunk 1]
```diff
@@ -135,7 +135,7 @@ std::vector<std::vector<size_t>> CoffedSparseMatrix::non_zero_columns(){
         auto& row = rows[row_num];
         std::vector<size_t> el;
         el.reserve(row.size());
-        for (auto& it = row.cbegin(); it != row.cend(); it++){
+        for (auto it = row.cbegin(); it != row.cend(); it++){
```
Comment (mishana) [inline src/corona_hakab_model/parasymbolic_matrix/parasymbolic.cpp:138]: better use auto&&

Comment (bentheiii) [reply src/corona_hakab_model/parasymbolic_matrix/parasymbolic.cpp:138]: agreed

Comment (benglewis) [reply src/corona_hakab_model/parasymbolic_matrix/parasymbolic.cpp:138]: OK, does someone want to explain that to me?

Comment (benglewis) [reply src/corona_hakab_model/parasymbolic_matrix/parasymbolic.cpp:138]: @mishana Thank you for the explanation :)

**Pull request merged**